src/components/FiscalYear/FiscalYearDetails/FiscalYearInformation/FiscalYearInformation.js

```diff
@@ -128,11 +141,19 @@ FiscalYearInformation.propTypes = {
   name: PropTypes.string.isRequired,
   periodEnd: PropTypes.string.isRequired,
   periodStart: PropTypes.string.isRequired,
+  currency: PropTypes.string,
+  allocated: PropTypes.number,
+  unavailable: PropTypes.number,
+  available: PropTypes.number,
 };
 
 FiscalYearInformation.defaultProps = {
   acqUnitIds: [],
   description: '',
+  currency: '',
```

Comment:
Currency as empty string makes not too much sense. IMHO, it's better to leave it as is